plugins/poll/app/serializers/poll_serializer.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class PollSerializer < ApplicationSerializer
-  attributes :name,
+  attributes :id,
```

Comment:
What do you need the `id` for?

Reply:
การทำวงาน

Reply:
It's used in the serializer for IRV:

```
 def irv_outcome
    DiscoursePoll::Irv.irv_outcome(object.id)
 end
```

Reply:
I believe it's redundant now, good catch, removed: https://github.com/discourse/discourse/pull/27155/commits/e7bbb8b5cf9147dcc0a14db4e63d0638530071ba